docs-requirements.txt

```diff
@@ -3,3 +3,4 @@
 -r test-requirements.txt
 sphinx~=1.4.9
 sphinx-argparse~=0.1.15
+sphinx_rtd_theme
```

Comment:
@Mariatta This theme is mentioned in the "Building Documentation" section of  `Development Setup Notes`. I guess we should remove it now :)